velox/substrait/TypeUtils.cpp

```diff
@@ -58,6 +83,21 @@ TypePtr toVeloxType(const std::string& typeName) {
       return BIGINT();
     case TypeKind::UNKNOWN:
       return UNKNOWN();
+    case TypeKind::ROW: {
+      std::vector<std::string> structTypeNames;
```

Comment:
This could would be easier to read if written as 

```
auto structTypeNames = getTypesFromCompoundName(typeName, structTypeNames);
```

Reply:
> This could would be easier to read if written as
> 
> ```
> auto structTypeNames = getTypesFromCompoundName(typeName, structTypeNames);
> ```

its fixed

Reply:
> > This could would be easier to read if written as
> > ```
> > auto structTypeNames = getTypesFromCompoundName(typeName, structTypeNames);
> > ```
> 
> its fixed.